src/test/java/me/devyonghee/racingcar/model/RandomMovementPolicyTest.java

```diff
@@ -0,0 +1,40 @@
+package me.devyonghee.racingcar.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@DisplayName("랜덤 이동 정책")
+class RandomMovementPolicyTest {
+
+    @Test
+    @DisplayName("객체화")
+    void instance() {
+        assertThatNoException().isThrownBy(() -> RandomMovementPolicy.from(new Random()));
+    }
+
+    @Test
+    @DisplayName("랜덤 객체는 필수")
+    void instance_nullRandom_thrownIllegalArgumentException() {
+        assertThatIllegalArgumentException().isThrownBy(() -> RandomMovementPolicy.from(null));
+    }
+
+    @ParameterizedTest(name = "[{index}] 랜덤한 수가 {0} 이라면 이동은 {1}")
+    @DisplayName("랜덤한 숫자가 4이상이면 이동")
+    @CsvSource({"3,STOP", "4,MOVE", "5,MOVE"})
+    void movement(int randomNumber, Movement expected) {
+        //given
+        Random random = mock(Random.class);
+        when(random.nextInt(anyInt())).thenReturn(randomNumber);
+        //when, then
+        assertThat(RacingCar.from(RandomMovementPolicy.from(random)).movement()).isEqualTo(expected);
```

Comment:
다음과 같이 람다를 활용하는 방법도 참고해주세요~
mock을 따로 구현하지 않아도 가능할 것 같습니다.
```suggestion
        assertThat(RacingCar.from(() -> Movement.MOVE).movement()).isEqualTo(expected);
```

Reply:
코멘트 감사합니다! 🙇‍♂️
그런데 이 테스트는 `RacingCar` 가 아니라 `MovementPolicy` 에 대한 테스트였는데 제가 실수했네요
그래서 람다보다 `Random` 을 그대로 mock 처리한 테스트로 유지하겠습니다!